src/dev/tslint/pick_files_to_lint.ts

```diff
@@ -22,5 +22,5 @@ import { ToolingLog } from '@kbn/dev-utils';
 import { File } from '../file';
 
 export function pickFilesToLint(log: ToolingLog, files: File[]) {
-  return files.filter(file => file.isTypescript());
+  return files.filter(file => file.isTypescript() && !file.isFixture());
```

Comment:
The issue this is that the fixture files are excluded from our main TS project because they can (and in my case need to) include invalid typescript for the purpose of making sure our types impose certain limitations. Since the files are not a part of a TS project they fail the pre-commit hook since TSLint needs the project file for a specific file to run the type-aware checks.

It's possible that we could run fixtures through a linting phase that isn't type enabled, but I'd rather do that in a separate PR. See https://github.com/elastic/kibana/issues/20910

Reply:
Ah, got it, thanks for explanation.